### DISCUSSION TITLE
difference between launchpad and other coordinator firmware for CC2652P Sonoff Dongle

### DISCUSSION THREAD

**Discussion by tivo65** [Koenkk/Z-Stack-firmware#519] [Q&A]
Basic question, what is the difference between these two firmware files? - other vs. launchpad.  I have 3 of the - ZBdongle-P Based on TI CC2652P + CP2102N chips (one coordinator and two routers).  Should I be using the launchpad version?  the router firmware I flashed a few months ago was called launchpad ver 20221102  

CC1352P2_CC2652P_launchpad_coordinator_20240710.zip
vs.
CC1352P2_CC2652P_other_coordinator_20240710.zip

**Comment by Koenkk** [comment]
Refer to https://www.zigbee2mqtt.io/guide/adapters/zstack.html for the correct firmware for your adapter.

**Reply by tivo65** [reply]
thanks for the reply.  I had already read the materials on that link - but I cannot see anywhere in that document where is uses the word "launchpad" vs. "other" or outlines suggested firmware.  I can see my dongle listed on the recommend list, but don't see where it outlines the correct firmware.  (in times past I think I remember a link from this site to the recommended firmware for the listed dongle.)

**Reply by Koenkk** [reply]
Click on coordinator firmware (dongle P uses the launchpad one)

<img width="688" alt="Screenshot 2024-09-10 at 14 46 22" src="https://github.com/user-attachments/assets/074c6159-1f5b-4a9a-8c0a-86d624c7faa7">

**Reply by rct** [reply]
Last night. I probably spent 10+ minutes going in circles, not realizing those were collapsed sections.. It was late, my tired old eyes were not noticing the tiny triangles, they looked too much like bullets.

Suggestion:  Can you please style those links to make it clearer that they are sections?   Maybe a big table would be clearer?  Or, Style that text with a different color to make it clearer they are a click target or use a bigger pointer triangle, also maybe a different color than the text?

**Reply by tivo65** [reply]
wow, I did not realize there are collapsed sections, did not realize those triangles had more info.  rct, you have me beat because I spent more than 10 minutes looking for the link to download.  thanks Koenkk for taking the time to answer my question.

**Reply by tivo65** [reply]
I also realized that there must be a problem with my browser (mac using Brave which is chrome based) where the triangles do not always show up.  
<img width="785" alt="Screenshot 2024-09-10 at 3 54 52 PM" src="https://github.com/user-attachments/assets/0e1d1e65-94c1-4dd3-8812-11c1df3e2d09">

**Reply by JulienDevillers** [reply]
Hello,
This does not explain the difference between "launchpad" and "other". I'm building a custom device based on CC2652P and it would be very interesting for me to understand. It could orient my design.
Thanks.

**Comment by Koenkk** [comment]
Please review https://github.com/Koenkk/zigbee2mqtt.io/pull/3016

**Reply by tivo65** [reply]
thanks!
